src/components/item/Comment.tsx

```diff
@@ -1,8 +1,12 @@
 import './Comment.css';
 import avatar from '../../assets/avatar.svg';
 import { formatDateToTimeAgo } from '../../utils/dateFormat';
+import { Comment as CommentData } from '../../services/itemApiTypes';
 
-export default function Comment({ content, createdAt, writer }) {
+interface CommentProps
+  extends Pick<CommentData, 'id' | 'content' | 'createdAt' | 'writer'> {}
```

Comment:
P2:
`Commnet`에서 `updateAt`을 제외한 나머지가 필요하니 아래처럼 하셔도 좋을 것 같습니다.

```suggestion
interface CommentProps
  extends Omit<CommentData, 'updatedAt'> {}
```